.dotnet/src/Custom/Common/GeneratorStubs.cs

```diff
@@ -9,4 +9,7 @@ internal interface IInternalListResponse<T>
     string FirstId { get; }
     string LastId { get; }
     bool HasMore { get; }
-}
\ No newline at end of file
+}
+
+[CodeGenModel("AssistantToolsFileSearchTypeOnly")]
+internal readonly partial struct InternalAssistantToolsFileSearchTypeOnly { }
```

Comment:
Should this be in OpenAI.Assistants namespace?

Reply:
Good catch. Yes.